.travis.yml

```diff
@@ -1,23 +1,21 @@
-sudo: required
-dist: trusty
-# Force travis to use its minimal image with default Python settings
+sudo: required 
+dist: trusty 
 language: generic
 compiler:
```

Comment:
I think we can either have language specific tags and if the language tag is generic then compiler parameter can be omitted.